pkg/session/session.go

```diff
@@ -3398,6 +3424,10 @@ func bootstrapSessionImpl(store kv.Storage, createSessionsImpl func(store kv.Sto
 	if err != nil {
 		return nil, err
 	}
+	err = InitTiDBSchemaCacheSize(store)
+	if err != nil {
+		logutil.BgLogger().Fatal("init tidb schema cache size error", zap.Error(err))
```

Comment:
```suggestion
		return nil, err
```

The caller should be able to handle this.